test/server_integration/http/ssl/config.js

```diff
@@ -9,13 +9,15 @@
 
 import { readFileSync } from 'fs';
 import { CA_CERT_PATH, KBN_CERT_PATH, KBN_KEY_PATH } from '@kbn/dev-utils';
+import { ScoutTestRunConfigCategory } from '@kbn/scout-info';
 import { createKibanaSupertestProvider } from '../../services';
 
 export default async function ({ readConfigFile }) {
   const httpConfig = await readConfigFile(require.resolve('../../config.base.js'));
   const certificateAuthorities = [readFileSync(CA_CERT_PATH)];
 
   return {
+    testConfigCategory: ScoutTestRunConfigCategory.UI_TEST,
```

Comment:
Not sure about this one. These tests (and P12 also) are not really UI, but they are not API or unit tests either. I think we can agree to group them with UI tests, otherwise I think we'd need another category.

Reply:
As discussed in slack, I'll merge as is and we can introduce a new category in a follow-up if we think it's needed.